### PR TITLE
PR addressing issue958 

### PR DESCRIPTION
**Pull request by riccardoAlbertoni**
Adding missing language tags to WD examples #958

### CODE REVIEW

File: dcat/examples/csiro-dap-examples.ttl [hunk 1]
```diff
@@ -232,13 +232,13 @@ dap:atnf-P366-2003SEPT_1
       rdf:type adms:Identifier ;
       dct:creator <https://www.doi.org/> ;
       skos:notation "10.4225/08/598dc08d07bb7" ;
-      adms:schemeAgency "International DOI Foundation" ;
+      adms:schemeAgency "International DOI Foundation"en ;
```
Comment (riccardoAlbertoni) [inline dcat/examples/csiro-dap-examples.ttl]:
```suggestion
      adms:schemeAgency "International DOI Foundation"@en ;
```

**Comment by riccardoAlbertoni** [comment]
I have added the language tags in all the examples in the DCAT WD except when  IMO adding the language tag does not make much sense, such as  
-  rdf:value in example 31.
-  rdf:value and XMLLiteral in examples 39 and 40 -
- dct:type "Observation"  and dct:bibliographicCitation in example 46 

I have also added language tags to TTL file contained in the directory of examples. 
Please @dr-shorthair check if this makes sense in your examples.  I guess the json-ld examples were not manually curated and so we can obtain the update simply converting from the TTL.  Is this ok?

File: dcat/examples/csiro-dap-examples.ttl [hunk 2]
```diff
@@ -232,13 +232,13 @@ dap:atnf-P366-2003SEPT_1
       rdf:type adms:Identifier ;
       dct:creator <https://www.doi.org/> ;
       skos:notation "10.4225/08/598dc08d07bb7" ;
-      adms:schemeAgency "International DOI Foundation" ;
+      adms:schemeAgency "International DOI Foundation"@en ;
```
Comment (dr-shorthair) [inline dcat/examples/csiro-dap-examples.ttl]: This is a proper name so I don't think it should have a language-tag

File: dcat/examples/csiro-dap-examples.ttl [hunk 3]
```diff
@@ -361,21 +361,21 @@ dap:d33937
...
-  dcat:keyword "OWL" ;
-  dcat:keyword "OWL-Time" ;
-  dcat:keyword "RDF" ;
-  dcat:keyword "SOSA" ;
-  dcat:keyword "SSN" ;
-  dcat:keyword "geologic timescale" ;
-  dcat:keyword "reference system" ;
-  dcat:keyword "stratigraphy" ;
-  dcat:keyword "vocabulary" ;
+  dcat:keyword "GeoSPARQL"@en ;
```
Comment (dr-shorthair) [inline dcat/examples/csiro-dap-examples.ttl]: Several of these are acronyms or proper names, so not language-specific

File: dcat/examples/csiro-dap-examples.ttl [hunk 4]
```diff
@@ -548,16 +548,16 @@ dap:d33937_1
...
```
Comment (dr-shorthair) [inline dcat/examples/csiro-dap-examples.ttl]: Proper names and acronyms

File: dcat/examples/csiro-dap-examples.ttl [hunk 5]
```diff
@@ -568,5 +568,5 @@ dap:doi-P366-2003SEPT
   rdf:type adms:Identifier ;
   dct:creator <https://researchdata.ands.org.au/> ;
   skos:notation "10.4225/08/598dc08d07bb7" ;
-  adms:schemeAgency "International DOI Foundation" ;
+  adms:schemeAgency "International DOI Foundation"@en ;
```
Comment (dr-shorthair) [inline dcat/examples/csiro-dap-examples.ttl]: proper name

File: dcat/examples/ga-courts.ttl [hunk 6]
```diff
@@ -18,28 +18,28 @@
...
+  rdfs:label "GA Judicial Courts"@en;
   owl:imports <http://www.w3.org/2006/vcard/ns> ;
   owl:imports <http://www.w3.org/ns/dcat> ;
 .
 ga-courts:GA
   rdf:type v:Organization ;
-  rdfs:label "Commonwealth of Australia (Geoscience Australia)" ;
-  v:country-name "Australia" ;
+  rdfs:label "Commonwealth of Australia (Geoscience Australia)"@en;
+  v:country-name "Australia"@en;
```
Comment (dr-shorthair) [inline dcat/examples/ga-courts.ttl]: I don't think the values in a vCard are language-specific strings

File: dcat/examples/ga-courts.ttl [hunk 7]
```diff
@@ -128,26 +128,26 @@ ga-courts:jc-wfs
...
   dcat:endpointURL <http://services.ga.gov.au/gis/services/Judicial_Courts/MapServer/WFSServer> ;
-  dcat:keyword "Australia" ;
-  dcat:keyword "Published_External" ;
-  dcat:keyword "WFS" ;
-  dcat:keyword "law courts" ;
-  dcat:keyword "web feature service" ;
-  dcat:keyword "web service" ;
+  dcat:keyword "Australia"@en;
+  dcat:keyword "Published_External"@en;
+  dcat:keyword "WFS"@en;
```
Comment (dr-shorthair) [inline dcat/examples/ga-courts.ttl]: Acronym

File: dcat/examples/ga-courts.ttl [hunk 8]
```diff
@@ -95,27 +95,27 @@ ga-courts:jc-esri
...
   dcat:endpointDescription <http://services.ga.gov.au/gis/services/Judicial_Courts/MapServer/WMSServer?request=GetCapabilities&service=WMS> ;
   dcat:endpointURL <http://services.ga.gov.au/gis/rest/services/Judicial_Courts/MapServer> ;
-  dcat:keyword "Australia" ;
-  dcat:keyword "MapServer" ;
-  dcat:keyword "Published_External" ;
-  dcat:keyword "law courts" ;
-  dcat:keyword "web mapping" ;
-  dcat:keyword "web service" ;
+  dcat:keyword "Australia"@en;
+  dcat:keyword "MapServer"@en;
```
Comment (dr-shorthair) [inline dcat/examples/ga-courts.ttl]: Proper name

File: dcat/examples/ga-courts.ttl [hunk 9]
```diff
@@ -77,15 +77,15 @@ ga-courts:jc
...
-  dcat:keyword "GIS" ;
-  dcat:keyword "National dataset" ;
-  dcat:keyword "Published_External" ;
-  dcat:keyword "culture" ;
-  dcat:keyword "infrastructure" ;
-  dcat:keyword "risk analysis" ;
-  dcat:keyword "risk assessment" ;
+  dcat:keyword "AU"@en;
+  dcat:keyword "Database"@en;
+  dcat:keyword "GIS"@en;
```
Comment (dr-shorthair) [inline dcat/examples/ga-courts.ttl]: Acronym

File: dcat/examples/ga-courts.ttl [hunk 10]
```diff
@@ -167,18 +167,18 @@ ga-courts:jc-wms
...
   dcat:endpointURL <http://services.ga.gov.au/gis/services/Judicial_Courts/MapServer/WMSServer> ;
-  dcat:keyword "Australia" ;
-  dcat:keyword "Published_External" ;
-  dcat:keyword "WMS" ;
-  dcat:keyword "law courts" ;
-  dcat:keyword "web map service" ;
-  dcat:keyword "web service" ;
+  dcat:keyword "Australia"@en;
+  dcat:keyword "Published_External"@en;
+  dcat:keyword "WMS"@en;
```
Comment (dr-shorthair) [inline dcat/examples/ga-courts.ttl]: Acronym

File: dcat/examples/ga-wms.ttl [hunk 11]
```diff
@@ -16,24 +16,24 @@
...
-  dct:title "Digital Elevation Model (DEM) of Australia derived from STRM with 1 Second Grid - Hydrologically Enforced" ;
+  dct:title "Digital Elevation Model (DEM) of Australia derived from STRM with 1 Second Grid - Hydrologically Enforced"@en ;
   dct:type dctype:Service ;
   dct:type <https://inspire.ec.europa.eu/metadata-codelist/SpatialDataServiceType/view> ;
   dcat:endpointDescription <http://services.ga.gov.au/gis/services/DEM_SRTM_1Second_Hydro_Enforced/MapServer/WMSServer?request=GetCapabilities&service=WMS> ;
   dcat:endpointURL <http://services.ga.gov.au/gis/services/DEM_SRTM_1Second_Hydro_Enforced/MapServer/WMSServer> ;
-  dcat:keyword "DEM" ;
-  dcat:keyword "ECOLOGY Landscape" ;
-  dcat:keyword "LAND Topography Models" ;
+  dcat:keyword "DEM"@en ;
```
Comment (dr-shorthair) [inline dcat/examples/ga-wms.ttl]: Acronym

**Review by dr-shorthair — CHANGES_REQUESTED** [review]

**Comment by makxdekkers** [comment]
@riccardoAlbertoni I think "Observation" as value for dct:type in example 46 is incorrect. Object of dct:type should not be a literal.

File: dcat/examples/csiro-dap-examples.ttl (hunk 2 above)
Comment (riccardoAlbertoni) [reply dcat/examples/csiro-dap-examples.ttl]:
```suggestion
      adms:schemeAgency "International DOI Foundation";
```

File: dcat/examples/csiro-dap-examples.ttl (hunk 3 above)
Comment (riccardoAlbertoni) [inline dcat/examples/csiro-dap-examples.ttl]:
```suggestion
  dcat:keyword "GeoSPARQL" ;
```

File: dcat/examples/csiro-dap-examples.ttl [hunk 12]
```diff
@@ -361,21 +361,21 @@ dap:d33937
...
-  dcat:keyword "OWL-Time" ;
-  dcat:keyword "RDF" ;
-  dcat:keyword "SOSA" ;
-  dcat:keyword "SSN" ;
-  dcat:keyword "geologic timescale" ;
-  dcat:keyword "reference system" ;
-  dcat:keyword "stratigraphy" ;
-  dcat:keyword "vocabulary" ;
+  dcat:keyword "GeoSPARQL"@en ;
+  dcat:keyword "OWL"@en ;
```
Comment (riccardoAlbertoni) [inline dcat/examples/csiro-dap-examples.ttl]:
```suggestion
  dcat:keyword "OWL" ;
```

File: dcat/examples/csiro-dap-examples.ttl [hunk 13]
```diff
@@ -361,21 +361,21 @@ dap:d33937
...
+  dcat:keyword "OWL-Time"@en ;
```
Comment (riccardoAlbertoni) [inline dcat/examples/csiro-dap-examples.ttl]:
```suggestion
  dcat:keyword "OWL-Time" ;
```

File: dcat/examples/csiro-dap-examples.ttl [hunk 14]
```diff
@@ -361,21 +361,21 @@ dap:d33937
...
+  dcat:keyword "RDF"@en ;
```
Comment (riccardoAlbertoni) [inline dcat/examples/csiro-dap-examples.ttl]:
```suggestion
  dcat:keyword "RDF" ;
```

File: dcat/examples/csiro-dap-examples.ttl [hunk 15]
```diff
@@ -361,21 +361,21 @@ dap:d33937
...
+  dcat:keyword "SOSA"@en ;
```
Comment (riccardoAlbertoni) [inline dcat/examples/csiro-dap-examples.ttl]:
```suggestion
  dcat:keyword "SOSA" ;
```

File: dcat/examples/csiro-dap-examples.ttl [hunk 16]
```diff
@@ -361,21 +361,21 @@ dap:d33937
...
+  dcat:keyword "SSN"@en ;
```
Comment (riccardoAlbertoni) [inline dcat/examples/csiro-dap-examples.ttl]:
```suggestion
  dcat:keyword "SSN" ;
```

File: dcat/examples/csiro-dap-examples.ttl (hunk 4 above)
Comment (riccardoAlbertoni) [reply dcat/examples/csiro-dap-examples.ttl]:
```suggestion
  dcat:keyword "GeoSPARQL" ;
```

File: dcat/examples/csiro-dap-examples.ttl [hunk 17]
```diff
@@ -548,16 +548,16 @@ dap:d33937_1
...
-  dcat:keyword "OWL-Time" ;
-  dcat:keyword "RDF" ;
-  dcat:keyword "SOSA" ;
-  dcat:keyword "SSN" ;
-  dcat:keyword "geologic timescale" ;
-  dcat:keyword "reference system" ;
-  dcat:keyword "stratigraphy" ;
-  dcat:keyword "vocabulary" ;
+  dcat:keyword "GeoSPARQL"@en ;
+  dcat:keyword "OWL"@en ;
```
Comment (riccardoAlbertoni) [inline dcat/examples/csiro-dap-examples.ttl]:
```suggestion
  dcat:keyword "OWL" ;
```

File: dcat/examples/csiro-dap-examples.ttl [hunk 18]
```diff
@@ -548,16 +548,16 @@ dap:d33937_1
...
+  dcat:keyword "OWL-Time"@en ;
```
Comment (riccardoAlbertoni) [inline dcat/examples/csiro-dap-examples.ttl]:
```suggestion
  dcat:keyword "OWL-Time" ;
```

File: dcat/examples/csiro-dap-examples.ttl [hunk 19]
```diff
@@ -548,16 +548,16 @@ dap:d33937_1
...
+  dcat:keyword "RDF"@en ;
```
Comment (riccardoAlbertoni) [inline dcat/examples/csiro-dap-examples.ttl]:
```suggestion
  dcat:keyword "RDF" ;
```

File: dcat/examples/csiro-dap-examples.ttl [hunk 20]
```diff
@@ -548,16 +548,16 @@ dap:d33937_1
...
+  dcat:keyword "SOSA"@en ;
```
Comment (riccardoAlbertoni) [inline dcat/examples/csiro-dap-examples.ttl]:
```suggestion
  dcat:keyword "SOSA" ;
```

File: dcat/examples/csiro-dap-examples.ttl [hunk 21]
```diff
@@ -548,16 +548,16 @@ dap:d33937_1
...
+  dcat:keyword "SSN"@en ;
```
Comment (riccardoAlbertoni) [inline dcat/examples/csiro-dap-examples.ttl]:
```suggestion
  dcat:keyword "SSN" ;
```

File: dcat/examples/csiro-dap-examples.ttl (hunk 5 above)
Comment (riccardoAlbertoni) [reply dcat/examples/csiro-dap-examples.ttl]:
```suggestion
  adms:schemeAgency "International DOI Foundation" ;
```

File: dcat/examples/ga-courts.ttl (hunk 6 above)
Comment (riccardoAlbertoni) [reply dcat/examples/ga-courts.ttl]:
```suggestion
  v:country-name "Australia" ;
```

File: dcat/examples/ga-courts.ttl (hunk 9 above)
Comment (riccardoAlbertoni) [reply dcat/examples/ga-courts.ttl]:
```suggestion
  dcat:keyword "GIS" ;
```

File: dcat/examples/ga-courts.ttl [hunk 22]
```diff
@@ -77,15 +77,15 @@ ga-courts:jc
...
-  dcat:keyword "AU" ;
-  dcat:keyword "Database" ;
-  dcat:keyword "GIS" ;
-  dcat:keyword "National dataset" ;
-  dcat:keyword "Published_External" ;
-  dcat:keyword "culture" ;
-  dcat:keyword "infrastructure" ;
-  dcat:keyword "risk analysis" ;
-  dcat:keyword "risk assessment" ;
+  dcat:keyword "AU"@en;
```
Comment (riccardoAlbertoni) [inline dcat/examples/ga-courts.ttl]:
```suggestion
  dcat:keyword "AU" ;
```

File: dcat/examples/ga-courts.ttl (hunk 8 above)
Comment (riccardoAlbertoni) [reply dcat/examples/ga-courts.ttl]:
```suggestion
  dcat:keyword "MapServer" ;
```

File: dcat/examples/ga-courts.ttl (hunk 7 above)
Comment (riccardoAlbertoni) [reply dcat/examples/ga-courts.ttl]:
```suggestion
  dcat:keyword "WFS" ;
```

File: dcat/examples/ga-courts.ttl (hunk 10 above)
Comment (riccardoAlbertoni) [reply dcat/examples/ga-courts.ttl]:
```suggestion
  dcat:keyword "WMS" ;
```

File: dcat/examples/ga-wms.ttl (hunk 11 above)
Comment (riccardoAlbertoni) [reply dcat/examples/ga-wms.ttl]:
```suggestion
  dcat:keyword "DEM" ;
```

File: dcat/index.html [hunk 23]
```diff
@@ -2993,25 +2993,25 @@ <h2>Dereferenceable identifiers</h2>
   # reading https://www.w3.org/TR/skos-reference/#notations more than one skos:notation can be set
   skos:notation "https://doi.org/10.5281/zenodo.1486279"^^xsd:anyURI;
   # the authority/agency defining the identifier scheme, used if the agency has no URI
-  adms:schemaAgency "International DOI Foundation" ;
+  adms:schemaAgency "International DOI Foundation"@en ;
```
Comment (riccardoAlbertoni) [inline dcat/index.html]:
```suggestion
  adms:schemaAgency "International DOI Foundation" ;
```

File: dcat/index.html [hunk 24]
```diff
@@ -2993,25 +2993,25 @@ <h2>Dereferenceable identifiers</h2>
...
   # the authority/agency defining the identifier scheme, used if the agency has URI
   dct:creator  ex:InternationalDOIFundation ;
   .
 
 ex:InternationalDOIFundation a foaf:Organization ;
-  rdfs:label "International DOI Foundation";
+  rdfs:label "International DOI Foundation"@en;
```
Comment (riccardoAlbertoni) [inline dcat/index.html]:
```suggestion
  rdfs:label "International DOI Foundation" ;
```

File: dcat/index.html [hunk 25]
```diff
@@ -2993,25 +2993,25 @@ <h2>Dereferenceable identifiers</h2>
...
   foaf:homepage &lt;https://www.doi.org/&gt; ;
   .
 
 &lt;https://example.org/PoelenJorritH&gt; a foaf:Person;
-  foaf:name "Jorrit H. Poelen" ;
+  foaf:name "Jorrit H. Poelen"@en ;
```
Comment (riccardoAlbertoni) [inline dcat/index.html]:
```suggestion
  foaf:name "Jorrit H. Poelen" ;
```

File: dcat/index.html [hunk 26]
```diff
@@ -2993,25 +2993,25 @@ <h2>Dereferenceable identifiers</h2>
...
   adms:identifier &lt;https://example.org/PoelenJorritHID&gt; ;
   .
 
 &lt;https://example.org/PoelenJorritHID&gt; a adms:Identifier;
   skos:notation "https://orcid.org/0000-0003-3138-4118"^^xsd:anyURI ;
   # the authority/agency defining the identifier scheme, used if the agency has no URI
-  adms:schemaAgency "ORCID" ;
+  adms:schemaAgency "ORCID"@en ;
```
Comment (riccardoAlbertoni) [inline dcat/index.html]:
```suggestion
  adms:schemaAgency "ORCID" ;
```

File: dcat/index.html [hunk 27]
```diff
@@ -3047,11 +3047,11 @@ <h2>Indicating common identifier types</h2>
   ...
   adms:identifier &lt;https://example.org/sid&gt; .
 
-  &lt;https://example.org/sid&gt; rdf:type adms:Identifier ;
+&lt;https://example.org/sid&gt; rdf:type adms:Identifier ;
   # the actual id
   skos:notation "PA 1-060-815"^^ex:type ;
   # Human readable schema agency
-  adms:schemaAgency "US Copyright Office" ;
+  adms:schemaAgency "US Copyright Office"@en ;
```
Comment (riccardoAlbertoni) [inline dcat/index.html]:
```suggestion
  adms:schemaAgency "US Copyright Office" ;
```

**Comment by dr-shorthair** [comment]
@makxdekkers I was responsible for that. 
Yes, a URI is always preferable, though `dct:type` is a `rdf:Property` so a literal is allowed. 

There are a lot of options that might be OK 

1. http://www.opengis.net/def/observationType/OGC-OM/2.0/OM_Observation
2. http://www.w3.org/ns/sosa/Observation 
3. http://purl.obolibrary.org/obo/BCO_0000003 
4. http://dbpedia.org/resource/Observation

Which would you prefer? Note that 2. and 3. are formal OWL Classes, so probably would be the value of rdf:type rather than dct:type, and therefore might not be appropriate here as they imply much narrower semantics. 1. is similarly tied to a specific UML model. So maybe dbpedia is best?

**Comment by makxdekkers** [comment]
No. 4, Dbpedia sounds good.

**Review by dr-shorthair — APPROVED** [review]

**Review by davebrowning — APPROVED** [review]

**Pull request merged**